app/src/JoinRoomCard.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
-import { Formik, Form, Field, ErrorMessage } from 'formik';
+import {ErrorMessage, Field, Form, Formik} from 'formik';
 import "./css/main.css";
+
 const firebase = require("firebase");
 const assert = require("assert")
 require("firebase/firestore");
 
 class JoinRoomCard extends React.Component {
   submitValues(values, setSubmitting) {
-    console.log("Joining Room:", values)
+    console.log(`Joining Room: ${values}`);
```

Comment:
wat dis

Reply:
Apparently JS string interpolation [uses backtick](https://dmitripavlutin.com/string-interpolation-in-javascript/)